service/service.go

```diff
@@ -70,14 +79,17 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 			return nil, err
 		}
 
-		elasticHTTPClient = dphttp.NewClientWithTransport(awsSignerRT)
+		esClient, esClientErr = dpEs.NewClient(dpEsClient.Config{
+			ClientLib: dpEsClient.GoElasticV710,
+			Address:   cfg.ElasticSearchAPIURL,
+			Transport: awsSignerRT,
+		})
+		if esClientErr != nil {
```

Comment:
Surely we would want to return error here so  application closes

Reply:
Good spot.